src/NuGet.Core/NuGet.Build.Tasks.Pack/Pack.targets

```diff
@@ -139,6 +139,51 @@ Copyright (c) .NET Foundation. All rights reserved.
           TaskParameter="TargetOutputs"
           ItemName="_ProjectReferences" />
     </MSBuild>
+
+    <MSBuild
+      Condition="'$(PackBuildOutput)' == 'true'"
```

Comment:
It would be nice if this property had symmetry with the other properties below:  `IncludeBuildOutput`, for example.